### DISCUSSION TITLE
Failed to start the project using java-agent

### DISCUSSION THREAD

**Discussion by WindIsClear** [grafana/pyroscope#2538] [Q&A]
When I access the service performance analysis as java-agent, I find the startup error: `Caused by: java.lang.UnsatisfiedLinkError: /tmp/deploy-pyroscope/libasyncProfiler-linux-x64-cf6c1cb2b76a1610ddcf9eda23ae704a8594e91e.so: /lib64/libc.so.6: version GLIBC_2.14'\'' not found (required by /tmp/deploy-pyroscope/libasyncProfiler-linux-x64-cf6c1cb2b76a1610ddcf9eda23ae704a8594e91e.so)
` , The service machine system I use is Centos6.9. Is there any way to solve it? Such as configuring environment variables or adding startup parameters....

**Comment by korniltsev** [comment]
Our async-profiler fork is built with centos7 which was released 9 years ago. I dont think it is very reasonable to support something older.
Does genuine async-profiler work?
What is your java version?

**Comment by WindIsClear** [comment]
genuine async-profiler does not work properly. After studying async-profiler source code, GLIBC library fails to execute performance analysis call server. In addition, my java version is 
`java version "1.8.0_144"
Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed mode)`

**Reply by korniltsev** [reply]
I think your best option is to update yo centos7
Next option could be to ask question in genuine async profiler repo(although I doubt any luck there)
